editor/src/clj/editor/code/data.clj

```diff
@@ -501,7 +501,7 @@
 
 (defn tab-stops [glyph-metrics tab-spaces]
   (let [^double space-width (char-width glyph-metrics \space)
-        tab-width (* space-width (double tab-spaces))]
+        tab-width (max 1 (* space-width (double tab-spaces)))]
```

Comment:
Not sure why, but sometimes the width was reported as 0 which resulted in division by 0 where the tab-width was used